app/src/main/res/xml/appearance_settings.xml

```diff
@@ -57,4 +57,11 @@
         android:title="@string/tablet_mode_title"
         app:singleLineTitle="false"
         app:iconSpaceReserved="false" />
+
+    <PreferenceScreen
+        android:fragment="org.schabi.newpipe.settings.NotificationSettingsFragment"
+        android:summary="@string/settings_category_notification_summary"
+        android:title="@string/settings_category_notification_title"
+        app:iconSpaceReserved="false" />
+
```

Comment:
Uhm why is this here?

We already have the fragment in the main-settings.
Should the main-settings fragment get removed instead? (I would prefer that)

Reply:
That's the player notification. Not the new streams notification

Reply:
Yes. But now it's in the settings menu twice:
* in the "main"-settings
* in the "appearance"-settings

Reply:
Okay so this seems to be deleted in the ``debug`` ``main_settings.xml`` but not in the ``release`` variant.
Because of that I got confused.

I will remove it from the release variant, then everything should be fine.